sdk/datamigration/Azure.ResourceManager.DataMigration/samples/Generated/Samples/Sample_SqlMigrationServiceResource.cs

```diff
@@ -115,7 +115,7 @@ public async Task Update_UpdateSQLMigrationService()
         public async Task GetAuthKeys_RetrieveTheListOfAuthenticationKeys()
         {
             // Generated from example definition: specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2022-03-30-preview/examples/ListAuthKeysMigrationService.json
-            // this example is just showing the usage of "SqlMigrationServices_ListAuthKeys" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "SqlMigrationServices_listAuthKeys" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `list`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2023-07-15-preview/sqlmigration.json#L1747